src/main/java/sirius/web/dispatch/AssetsDispatcher.java

```diff
@@ -179,6 +186,14 @@ private boolean handleUnmodified(Template template, Response response) {
         return response.handleIfModifiedSince(template.getCompilationTimestamp());
     }
 
+    private boolean handleUnmodified(URLConnection urlConnection, Response response) {
+        return response.handleIfModifiedSince(urlConnection.getLastModified());
+    }
+
+    private boolean handleUnmodified(File file, Response response) {
+        return response.handleIfModifiedSince(file.lastModified());
+    }
```

Comment:
**Optional:** In my opinion these don't really have a benefit as they just shadow a call and use the same number of arguments, thus they don't simplify the code